### DISCUSSION TITLE
how to convert .py to .exe for customtkinter

### DISCUSSION THREAD

**Discussion by plugeit** [TomSchimansky/CustomTkinter#343] [General]
If it is giving error try to paste files into the output folder of auto-py-to-exe or you can add it in additional files section 
what files you have to paste?
files from python_version(python39/python310/etc.)/Lib/Site-packages/
ex: C:\Users\Dell\AppData\Local\Programs\Python\Python39\Lib\site-packages
here you will find two folders named customtkinter and cutomtkinter dist info copy them and paste it in the output folder it may work for you because it had worked for me.
![image](https://user-images.githubusercontent.com/72407828/182189449-133c081b-73e4-411f-8658-31f6a659c168.png)
![image](https://user-images.githubusercontent.com/72407828/182189511-d365ef31-45eb-47b5-b2a5-f07874471dd6.png)

**Comment by cho-amh-programmerExpert** [comment]
Thanks man for the help. It was taking me a long  time to find the issue. And now for the other Packages I can do the same thing
